### ISSUE TITLE
Ubuntu updater running very long

### ISSUE THREAD

**Issue by domcar** [quay/clair#453]
Hello,

I noticed that the ubuntu updater runs the all day and it causes 100% CPU usage on the server.
I had a look at clair logs and it seems like it takes alway a very long time to finish the job:

```
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"ubuntu.go:88","Time":"2017-07-31 20:52:11.009955","package":"Ubuntu"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"debian.go:63","Time":"2017-07-31 20:52:11.010095","package":"Debian"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"rhel.go:92","Time":"2017-07-31 20:52:11.010273","package":"RHEL"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"oracle.go:119","Time":"2017-07-31 20:52:11.010095","package":"Oracle Linux"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"alpine.go:52","Time":"2017-07-31 20:52:11.024586","package":"Alpine"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-07-31 20:52:11.771905","updater name":"alpine"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-07-31 20:52:12.640172","updater name":"rhel"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-07-31 20:52:13.337293","updater name":"oracle"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-07-31 20:52:25.714920","updater name":"debian"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-08-01 04:00:04.888603","updater name":"ubuntu"}


{"Event":"Start fetching vulnerabilities","Level":"info","Location":"oracle.go:119","Time":"2017-08-16 12:10:45.003645","package":"Oracle Linux"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"alpine.go:52","Time":"2017-08-16 12:10:45.003796","package":"Alpine"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"debian.go:63","Time":"2017-08-16 12:10:45.003848","package":"Debian"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"rhel.go:92","Time":"2017-08-16 12:10:45.003848","package":"RHEL"}
{"Event":"Start fetching vulnerabilities","Level":"info","Location":"ubuntu.go:88","Time":"2017-08-16 12:10:45.003752","package":"Ubuntu"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-08-16 12:10:45.791448","updater name":"alpine"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-08-16 12:10:48.550933","updater name":"oracle"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-08-16 12:10:50.095505","updater name":"debian"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-08-16 12:10:51.521762","updater name":"rhel"}
{"Event":"finished fetching","Level":"info","Location":"updater.go:227","Time":"2017-08-16 19:29:28.833025","updater name":"ubuntu"}
```
This is only an example but the logs show that it happens every day.

**Comment by hewen1125** [comment]
+1    "bzr so slow"

**Comment by jzelinskie** [comment]
Fixed in #568. Please open a new issue if you're still experiencing this bug on a non-bzr Clair.
